sdk/core/core-http/lib/serviceClient.ts

```diff
@@ -403,6 +405,8 @@ function createDefaultRequestPolicyFactories(credentials: ServiceClientCredentia
   if (credentials) {
     if (isRequestPolicyFactory(credentials)) {
       factories.push(credentials);
+    } else if (isTokenCredential(credentials)) {
+      factories.push(bearerTokenAuthenticationPolicy(credentials, '/.default'))
```

Comment:
I need to find a good way to pass in a scope here for default pipeline setup (or use a globally understood scope if there is such a thing?)  Any ideas?

Reply:
Scopes are totally defined by the service, I don't believe that there is a need to pass this into the client constructor.  In the case that the user believes he knows better what the scope should be they should either use their own auth policy, or their own implementation of TokenCredential which could ignore the passed in scope.  However, I struggle to come up with a scenario where this would work.

Reply:
Sounds good, thanks @schaabs!